### PR TITLE
feat: rework readme

### PR DESCRIPTION
**Pull request by jessicaschilling**
Closes https://github.com/ipfs-shipyard/ipfs-desktop/issues/1670.

- Rewrites README overall, primarily focusing on better explanation of features
- Updates throughout for outdatedness
- New screenshots
- Moved release checklist into DEVELOPER-NOTES.md to avoid wall of scary text
- Copy/clarity edits, etc etc

Note: Once this is approved/merged, I'll add some highlights (primarily features list) to https://docs.ipfs.io/install/ipfs-desktop/ in a separate PR in the docs repo.

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@lidel - do you mind a review? Thank you!

File: README.md [hunk 1]
```diff
@@ -1,105 +1,124 @@
...
+- **AppImage** (experimental): [ipfs-desktop-0.13.2-linux-x86_64.AppImage](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.13.2/ipfs-desktop-0.13.2-linux-x86_64.AppImage)\
+[![](https://img.shields.io/github/downloads/ipfs-shipyard/ipfs-desktop/v0.13.2/ipfs-desktop-0.13.2-linux-x86_64.AppImage.svg?style=flat-square&label=downloads)](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.13.2/ipfs-desktop-0.13.2-linux-x86_64.AppImage)
+- **FreeBSD** (experimental): [ipfs-desktop-0.13.2-linux-x64.freebsd](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.13.2/ipfs-desktop-0.13.2-linux-x64.freebsd)\
+[![](https://img.shields.io/github/downloads/ipfs-shipyard/ipfs-desktop/v0.13.2/ipfs-desktop-0.13.2-linux-x64.freebsd.svg?style=flat-square&label=downloads)](https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.13.2/ipfs-desktop-0.13.2-linux-x64.freebsd)
+- **Snapcraft** (community-maintained): `snap install ipfs-desktop`
+- **AUR** (maintained by [@alexhenrie](https://github.com/alexhenrie)): Use the [`ipfs-desktop` package](https://aur.archlinux.org/packages/ipfs-desktop/)
```
Comment (lidel) [inline README.md:112]: Not sure if this is up to date, @alexhenrie are you still maintaining ipfs-desktop package on AUR?
If not, we may need to remove this.

File: README.md [hunk 2]
```diff
@@ -109,116 +128,93 @@ npm run build
...
+
+### How does IPFS Desktop select the IPFS repo location?
+
+IPFS Desktop uses [ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl), which, by default, checks the `IPFS_PATH` environment variable. If that isn't set, it falls back to `$HOME/.ipfs`. As soon as the first run has succeded, repository location info is saved in the configuration file, which becomes the source of truth.
+
+To open your repo directory from the IPFS logo menu, select `Open Repository Directory` from the `Advanced` submenu.
+
+### Which version of IPFS does IPFS Desktop use?
+
+Since IPFS Desktop uses [ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl), it includes its own embedded IPFS binary, which in most circumstances is the latest version of [go-ipfs](https://github.com/ipfs/go-ipfs).
```
Comment (lidel) [inline README.md:202]: no need to bring ipfsd anymore, we define version directly in  `package.json`:

```suggestion
IPFS Desktop includes its own embedded binary in version defined in `package.json`, which iis the latest version of [go-ipfs](https://github.com/ipfs/go-ipfs) that passed QA for desktop use.
```

File: README.md [hunk 3]
```diff
@@ -109,116 +128,93 @@ npm run build
...
+
+You can check which version of IPFS you're running from the IPFS logo menu by looking in the `About` submenu.
+
+### Which flags does IPFS Desktop boot with?
+
+By default, IPFS Desktop starts the IPFS daemon with the flags `--migrate=true --routing=dhtclient ----enable-gc=true`. 
+
+You can change this in the config file by editing the `IPFS Config` section of IPFS Desktop's `Settings` screen.
```
Comment (lidel) [inline README.md:210]: This changed  when we decoupled Webui Settings from Desktop Settings:
```suggestion
You can change this in the Desktop config file by selecting `Open Configuration File` from the `Advanced` submenu.
```

**Review by lidel — CHANGES_REQUESTED** [review]
Thank you! This PR makes README much more useful.

I noticed some things were outdated, see small suggestions/fixes/clarifications inline.

File: README.md (hunk 1 above)
Comment (jessicaschilling) [reply README.md:112]: Looks like its last-updated date was 13 Oct, which is encouraging, but yes - @alexhenrie can you please confirm?

**Review by lidel — APPROVED** [review]

**Pull request merged**